core/common/src/CloudStorage.ts

```diff
@@ -116,7 +116,7 @@ export abstract class CloudStorageCache<TContentId, TContentType> {
       if (!request) {
         request = new Promise(async (resolve, reject) => {
           try {
-            container = await this.obtainContainerUrl(id, { name, resource });
+            container = await this.obtainContainerUrl(id, { name });
```

Comment:
Please explain this change in your PR description. @swbsi CloudStorageContainerDescriptor takes an optional `resource`. Is it correct to always omit it here?

Reply:
I believe I updated my PR description around the same time as this comment, does that description seem good to you?

Reply:
Explanation is good, thanks. I am concerned about breaking other implementations of this interface. Your description suggests we think we know that this change probably won't break current existing implementations that we know about but that's not super-reassuring.

What is the scope of the problem being solved - one or two extra GET requests?

Reply:
Yes thats exactly right "we might have to start up the backend to get the container URL...". This has shown to have quite a large performance impact. Even though the backend is being started up in the background on the first cache hit, there is still a lot of added time in figuring out when the backend is ready (even though it may have been ready for many seconds, but that is the readiness probe) and actually sending that request to the now stood up backend.